0017-app-standards.md

```diff
@@ -0,0 +1,116 @@
+
+## Decision
+
+The decision to work with ADRs is due to their entrenched nature within the
+Oasis ecosystem. Although adapting another project's process would be easy,
+having multiple proposal repositories could lead to confusion. ADRs also already
+have desiderata like process for community discussion and a flexible format.
+
+For use with application standards, ADRs shall have more structure to make
+contributions fair and straightforward. Specifically, clear requirements for
+moving promotion from draft to final, and additional required sections.
+
+Although community standards are only proposals, the _Decision_ section will
+keep its existing name for compatibility with the existing template. The
+decision in this case is to accept the standard for promotion.
+
+### Changes to the ADR template
+
+- add a new "Apps" component, which has the ParaTime as its sub-component
+
+### New Section Requirements
+
+#### Decision
+
+The _Decision_ section gets two new sub-sections:
+
+**Specification**: A complete description of the interface of the standard,
+including the threat/trust model, rationale for design decisions, alternative
+approaches, and references to related work. This section will generally be
+mostly prose with sprinkles of code for illustration.
+
+**Reference Implementation**: A basic implementation of the proposed standard
+written in a common language that targets the ParaTime runtime environment.
+The reference implementation in the ADR should be executable.
+
+#### Security Considerations
+
+This new section details any weak points of the proposal or common security
+flaws that a re-implementation of the specification may run into, as well as
+suggestions for avoiding security issues.
+
+### Acceptance Requirements
+
+Like all ADRs, an Apps component ADR will start as _Proposed_ and end up merged
+once _Accepted_. An application standard ADR following the above format will be
+accepted once:
+
+- there is consensus within the ParaTime's own community that the standard meets
+  its design goals
+- there are no outstanding compatibility or security issues
+- an ADR repo committer has signed off on the structure and format of the ADR
+
+## Alternatives
+
+One alternative is to fit the ParaTime-specific application standard proposals
+into the existing ADR template, but this would cause the _Decision_ section to
+become overloaded with the necessary information in an ad-hoc way.
+
+Another alternative is to encourage ParaTimes to do whatever they feel best.
+That's always allowed, of course, and it may sometimes be useful to wholesale
+copy the best practices of another community. However, if we make the ADR
+process convenient enough, the community can focus its collective effort on
+the one ADR repo.
+
+Within the chosen decision, there were many choices of structure from the now
+several EIP-like repos. The ones chosen were the minimum we need to get going,
+in the spirit of the lightweight ADR process.
```

Comment:
Maybe also explicitly mention that if a more specialized process is needed this can be improved later on and relevant ADRs transferred over.